<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1463</width>
    <height>784</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1441</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>401</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Tahoma&quot;;</string>
      </property>
      <property name="text">
       <string>Họ và tên</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>82</x>
        <y>20</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Tahoma&quot;;</string>
      </property>
      <property name="text">
       <string>Ngày sinh</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="ngay">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>31</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Tahoma&quot;;</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="thang">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Tahoma&quot;;</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="nam">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>60</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1999</number>
      </property>
      <property name="maximum">
       <number>2007</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Tahoma&quot;;</string>
      </property>
      <property name="text">
       <string>Giới tinh</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Nam</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nữ</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Tahoma&quot;;</string>
      </property>
      <property name="text">
       <string>Khối</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>A</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>C</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>D</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Tahoma&quot;;</string>
      </property>
      <property name="text">
       <string>Lớp</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Thêm</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>0</y>
       <width>641</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Tahoma&quot;;</string>
      </property>
      <property name="text">
       <string>File csv , txt</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>341</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Import</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>341</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Tahoma&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>(Lưu ý các file txt phải ngăn cách nhau bởi dấu | hoặc dấu ,)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 9pt &quot;Tahoma&quot;;</string>
      </property>
      <property name="text">
       <string>File csv , txt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="disconnect_CSDL">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>20</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Ngắt Kết Nối với CSDL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="connect_CSDL">
     <property name="geometry">
      <rect>
       <x>1260</x>
       <y>20</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Kết với với CSDL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refesh_CSDL">
     <property name="geometry">
      <rect>
       <x>1180</x>
       <y>90</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Làm mới CSDL </string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>1441</width>
      <height>521</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>STT</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Họ và Tên</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Giới tính</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ngày sinh</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Lớp</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hạnh kiểm </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Toán</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Vật Lý</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hóa</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tin</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sinh</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sử </string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Văn</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tiếng Anh</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Trung bình </string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
